core/corehttp/gateway_handler_unixfs__redirects.go

```diff
@@ -12,7 +12,7 @@ import (
 	files "github.com/ipfs/go-ipfs-files"
 	coreiface "github.com/ipfs/interface-go-ipfs-core"
 	ipath "github.com/ipfs/interface-go-ipfs-core/path"
-	"github.com/tj/go-redirects"
+	redirects "github.com/justincjohnson/go-ipfs-redirects"
```

Comment:
@justincjohnson mind making me an Admin (Owner?) in that repo? Seems only that level can move it between orgs.

(sadly under the water and I'll be reviewing things after Iceland, but we could move the repo before, so we have one thing less on our plate)

Reply:
@lidel from what I can tell I'm not able to make you an admin of a repo in my personal account.  Fission doesn't have more github seats currently or I'd do that right away.  Is there any reason not to just fork my repo into the ipfs org instead?  But yeah, I figured you had a ton on your plate.  I was hoping to meet you there but unfortunately can't make it anymore.  I hope you have a productive time there.

Reply:
Okay @lidel, instead of adding you as an admin I just transferred it to you.  You should see a request on your end.

Reply:
That did the trick! It is now at https://github.com/ipfs-shipyard/go-ipfs-redirects (you should still have access to it).
I also merged https://github.com/ipfs/specs/pull/289 and you should be able to finalize specs for final review.
Lmk if you are blocked on anything.
(Bummer that we can't meet in person, but I'm excited to work with you on landing both IPIP and this implementation after I'm back from Iceland)

Reply:
@lidel another gentle bump on this after Iceland, and I know you're busy...  any idea how soon you might have time to help get redirects support over the finish line? 🙏